src/js/components/Accordion/__tests__/__snapshots__/Accordion-test.tsx.snap

```diff
@@ -2515,7 +2515,7 @@ exports[`Accordion change to second Panel 2`] = `
             aria-hidden="false"
             class="c1 c13"
             open=""
-            style="max-height: 0;"
+            style="max-height: 0px; max-height: 0px;"
```

Comment:
this is the only file not sure why this changed? If you run `yarn jest -u` this came up?

Reply:
I went ahead and fixed this